nix/metal/packages/utm/default.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  undmg,
+  fetchurl,
+  stdenvNoCC,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "utm";
+  version = "4.2.0";
```

Comment:
Are you not using the one from nixpkgs (4.1.5) because you need 4.2.0?

Reply:
Yes, we needed the extra functionality/bug-fixes.  Also we should either pin this as it's own flake input or vendor pin it.  Since we needed the newer version, we've vendor pinned for now.